### PR TITLE
Fixes kudzu loosing mutations on harvest

### PR DESCRIPTION
**Pull request by Razharas**
Fixes kudzu loosing mutations on harvest
:cl: Razharas
rscadd: "Kudzu now keeps the mutations it had before and thus can be properly cultivated to be whatever you want"
/:cl:

### CODE REVIEW

**Comment by Supermichael777** [comment]
hah so breading oxgen vines WAS impossible

**Comment by Razharas** [comment]
Durning the one of the rewrites of the botanics someone broke it i suppose
Or maybe i forgot to make this work no idea

**Comment by Pyko1** [comment]
should actually make it possible to make spesswines mutate faster.

**Comment by Razharas** [comment]
Read the wiki hydroponics guide its all there how to tweak kudzu's properties

**Comment by Saegrimr** [comment]
Losing*

File: code/modules/hydroponics/grown/kudzu.dm [hunk 1]
```diff
@@ -35,6 +35,16 @@
 	new /obj/effect/spacevine_controller(user.loc, mutations, potency, production)
 	qdel(src)
 
+/obj/item/seeds/kudzu/Copy()
+	var/obj/item/seeds/kudzu/KZ = ..()
+	KZ.mutations = mutations.Copy()
```
Comment (phil235) [inline code/modules/hydroponics/grown/kudzu.dm]: maybe `return KZ` ?

**Pull request merged**